docs/examples/managed/zcpDemo.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        metadata={"version": "milvus 2.3", "year": "2023"},  # optional
</pre><p>could you show an example which enable metadata filter as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9605/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I've added example to apply metadata filters in as_query_engine